src/font/Atlas.zig

```diff
@@ -291,18 +291,17 @@ pub fn grow(self: *Atlas, alloc: Allocator, size_new: u32) Allocator.Error!void
     const data_old = self.data;
     const size_old = self.size;
 
-    // Allocate our new data
-    self.data = try alloc.alloc(u8, size_new * size_new * self.format.depth());
+    try self.nodes.ensureUnusedCapacity(1);
+    const data_new = try alloc.alloc(u8, size_new * size_new * self.format.depth());
+    errdefer comptime unreachable;
+
+    self.data = data_new;
     defer alloc.free(data_old);
-    errdefer {
-        alloc.free(self.data);
-        self.data = data_old;
-    }
 
     // Add our new rectangle for our added righthand space. We do this
     // right away since its the only operation that can fail and we want
     // to make error cleanup easier.
-    try self.nodes.append(alloc, .{
+    self.nodes.appendAssumeCapacity(alloc, .{
```

Comment:
`appendAssumeCapacity` doesn't need the alloc. Please make sure your code actually compiles before making a PR.

Reply:
I am not motivated enough to setup a local dev environment for testing, and I am fine with PR being closed for that reason :)

Reply:
> I'm guessing you didn't find this while exploring the code yourself

No, I just have a hobby of grepping zig projects for `errdefer`, as the code around is aways buggy: 

https://github.com/ziglang/zig/pull/22131

>The workflows were manually approved to run,

Yeah, I mean that if workflow didn't require manual approvals, I'd be able to iterate here quicker.